src/eventBody.js

```diff
@@ -27,7 +27,7 @@ export class EventBody {
 		this._context = {};
 
 		if (id) {
-			this._context.Id = id;
+			this._context.Id = id.toString();
```

Comment:
Were we planning on making this change to the Object and Target too?

Reply:
I think so, would future proof us in my opinion.